### PR TITLE
Fix update to use latest worker_pool

### PR DESCRIPTION
**Pull request by jagguli**

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -24,7 +24,7 @@
 %% == Dependencies ==
 
 {deps, [
-  {worker_pool, "3.1.0"},
+  {worker_pool, "5.1.0"},
```
Comment (elbrujohalcon) [inline rebar.config:27]:
```suggestion
  {worker_pool, "6.0.0"},
```

Latest worker pool is 6.0.0 now: https://hex.pm/packages/worker_pool/6.0.0

**Review by elbrujohalcon — CHANGES_REQUESTED** [review]
…and you need to update `rebar.lock`, too.

**Review by elbrujohalcon — CHANGES_REQUESTED** [review]
You still need to upgrade rebar.lock

Run `rebar3 upgrade --all` and push the changes

File: src/sumo_store.erl [hunk 2]
```diff
@@ -167,7 +167,7 @@ start_link(Name, Module, Options) ->
     {workers, PoolSize},
     {worker, {?MODULE, [Module, Options]}}
   ],
-  wpool:start_pool(Name, WPoolConfigOpts ++ WPoolOptions).
+  wpool:start_pool(Name, lists:ukeysort(1, WPoolConfigOpts ++ WPoolOptions)).
```
Comment (elbrujohalcon) [inline src/sumo_store.erl:170]: How will this work when you have the same key with different values in both lists? The idea is that the one in `WPoolConfigOpts` should override the one in `WPoolOptions`, I believe.

**Review by elbrujohalcon — CHANGES_REQUESTED** [review]

File: src/sumo_store.erl (hunk 2 above)
Comment (jagguli) [reply src/sumo_store.erl:170]: yes this removes any duplicates and only takes the first occurence of key in `[{key,value}]`.

```
1> lists:ukeysort(1,[{test, 1}, {test,2}]).
[{test,1}]
2> lists:ukeysort(1,[{test, 1}, {test,2}]++[{test,3},{test,4}]).
[{test,1}]
3> lists:ukeysort(1,[{test, 2}, {test,1}]++[{test,3},{test,4}]).
[{test,2}]
4> lists:ukeysort(1,[{test, 2}, {test,1}]++[{test,3},{test,4},{other,2}]).
[{other,2},{test,2}]
```

**Review by elbrujohalcon — APPROVED** [review]

**Pull request merged**